xtokens/src/tests.rs

```diff
@@ -431,6 +404,309 @@ fn send_as_sovereign_fails_if_bad_origin() {
 	});
 }
 
+#[test]
+fn para_transact_to_relay_remark_use_sovereign_account() {
+	Relay::execute_with(|| {
+		let _ = RelayBalances::deposit_creating(&para_a_account(), 6030);
+	});
+
+	ParaA::execute_with(|| {
+		parachain_transact_to_relaychian_remark();
+	});
+
+	Relay::execute_with(|| {
+		use relay::{Event, System};
+		assert!(System::events()
+			.iter()
+			.any(|r| matches!(r.event, Event::System(frame_system::Event::Remarked(_, _)))));
```

Comment:
Use `System::assert_has_event`, same to all other remark event checks.

Reply:
delete useless tests.